jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -15,210 +14,204 @@ pipeline {
-                    ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"${AWS_IP}" "export CORTX_RE_BRANCH=$CORTX_RE_BRANCH; git clone $CORTX_RE_REPO -b $CORTX_RE_BRANCH; pushd /home/centos/cortx-re/solutions/community-deploy; time sudo ./build-cortx.sh -b ${CORTX_TAG}"
-                popd
+                    export ROOT_PASSWORD=${ROOT_PASSWORD}
+                    terraform show -json terraform.tfstate | jq .values.outputs.aws_instance_private_dns.value 2>&1 | tee ec2_hostname.txt
+                    export HOST1=$(cat ec2_hostname.txt | jq '.[0]'| tr -d '",[]')
+                    export HOST2=$(cat ec2_hostname.txt | jq '.[1]'| tr -d '",[]')
+                    export HOST3=$(cat ec2_hostname.txt | jq '.[2]'| tr -d '",[]')
+                    export HOST4=$(cat ec2_hostname.txt | jq '.[3]'| tr -d '",[]')
+                    for ip in $PUBLIC_IP;do ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@${ip} "export ROOT_PASSWORD=$ROOT_PASSWORD && echo $ROOT_PASSWORD | sudo passwd --stdin root && git clone https://github.com/Seagate/cortx-re && pushd /home/centos/cortx-re/solutions/kubernetes && touch hosts && echo hostname=${HOST1},user=root,pass= > hosts && sed -i 's,pass=.*,pass=$ROOT_PASSWORD,g' hosts && echo hostname=${HOST2},user=root,pass= >> hosts && sed -i 's,pass=.*,pass=$ROOT_PASSWORD,g' hosts && echo hostname=${HOST3},user=root,pass= >> hosts && sed -i 's,pass=.*,pass=$ROOT_PASSWORD,g' hosts && echo hostname=${HOST4},user=root,pass= >> hosts && sed -i 's,pass=.*,pass=$ROOT_PASSWORD,g' hosts && cat hosts && sleep 120";done
+                    popd
             '''
             }
         }
-
-        stage ('EC2 connection prerequisites') {
+        stage('Execute cortx build script') {
             steps {
                 script { build_stage = env.STAGE_NAME }
-                sh label: 'Changing root password and creating hosts file', script: '''
+                sh label: 'Executing cortx build image script on Primary node', script: '''
                 pushd solutions/community-deploy/cloud/AWS
-                    export ROOT_PASSWORD=${ROOT_PASSWORD}              
-                    ssh -i cortx.pem -o StrictHostKeyChecking=no centos@"${AWS_IP}" "export ROOT_PASSWORD=$ROOT_PASSWORD && echo $ROOT_PASSWORD | sudo passwd --stdin root && pushd /home/centos/cortx-re/solutions/kubernetes && echo "'"hostname=$HOSTNAME,user=root,pass="'" > hosts && sed -i 's,pass=.*,pass=$ROOT_PASSWORD,g' hosts && cat hosts"
+                    export CORTX_RE_BRANCH=${CORTX_RE_BRANCH}
+                    export CORTX_TAG=${CORTX_TAG}
+                    sleep 120
+                    export PRIMARY_PUBLIC_IP=$(cat ip_public.txt | jq '.[0]'| tr -d '",[]')
+                    ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@${PRIMARY_PUBLIC_IP} "export CORTX_RE_BRANCH=$CORTX_RE_BRANCH; git clone $CORTX_RE_REPO -b $CORTX_RE_BRANCH; pushd /home/centos/cortx-re/solutions/community-deploy; time sudo ./build-cortx.sh"
```

Comment:
Need to pass `TAG` details to `build-cortx.sh`  Refer - https://github.com/Seagate/cortx-re/blob/main/jenkins/deployment/community/cortx-community-deploy.groovy#L104